app/controllers/pages_controller.rb

```diff
@@ -4,9 +4,8 @@ class PagesController < ApplicationController
   def home
     @registers = Register.available.all
 
-    @ready_registers = Register.available.where(register_phase: 'Beta')
-    @upcoming_registers = Register.available.where.not(register_phase: 'Beta')
-    @organisation_count = @ready_registers.distinct.count(:authority)
+    @registers_available = Register.available.where(register_phase: 'Beta')
```

Comment:
as we are repeating these queries now, we could DRY it up by moving them into the register model e.g.
```
  scope :organisation_count, lambda {
                               available
                                 .in_beta
                                 .distinct
                                 .count(:authority)
                             }
  scope :available_count, -> { available.in_beta.count }
```
Then in the template we can call them on the model e.g. `Register.organisation_count`